bubble_sort.rb

```diff
@@ -0,0 +1,32 @@
+def bubble_sort_by(array)
```

Comment:
This method should be named `bubble_sort` as this method is **not** handling `yield`.